addons/local-path-storage/0.0.22/install.sh

```diff
@@ -0,0 +1,49 @@
+
+function local-path-storage_join() {
+  local-path-storage_host_init
+}
+
+function local-path-storage() {
+    local src="$DIR/addons/local-path-storage/__releasever__"
+    local dst="$DIR/kustomize/local-path-storage"
+
+    local-path-storage_host_init
+
+    cp "$src/local-path-storage.yaml" "$dst/"
+
+    cp "$src/tmpl-kustomization.yaml" "$dst/"
+
+    if has_default_storageclass && ! local_path_storage_is_default_storageclass ; then
```

Comment:
should we make this a property of the addon so that its less racey? it can be true by default

Reply:
I just wonder if there is a better way to specify priority. This is a bit circumstantial as they happen to be in this order today and could be more explicit. I'll approve with it but we should think of a better way to support this "feature" in the future.